### PR TITLE
Add initial disqus comment support.

### LINKED ISSUE #23: Add comment system to races
Maybe disqus?

### PR DESCRIPTION
**Pull request by jyundt**
Closes #23

### CODE REVIEW

File: app/templates/race/details.html [hunk 1]
```diff
@@ -328,4 +328,26 @@ <h4 class="text-right">
...
   {%endif%}
 {%endif%}
+<div id="disqus_thread"></div>
+<script>
+
+/**
+*  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
+*  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables*/
+/*
+var disqus_config = function () {
```
Comment (johnwheffner) [inline app/templates/race/details.html]: I think this is supposed to be uncommented.

File: app/templates/race/details.html [hunk 2]
```diff
@@ -328,4 +328,26 @@ <h4 class="text-right">
...
+this.page.url = {{url_for('race.details',id=race.id)}};  // Replace PAGE_URL with your page's canonical URL variable
```
Comment (johnwheffner) [inline app/templates/race/details.html]: This is probably supposed to be a string enclosed in quotes.

Comment (johnwheffner) [reply app/templates/race/details.html]: Also I think we want _external=True

File: app/templates/race/details.html [hunk 3]
```diff
@@ -328,4 +328,26 @@ <h4 class="text-right">
...
+};
+*/
+(function() { // DON'T EDIT BELOW THIS LINE
+var d = document, s = d.createElement('script');
+s.src = 'https://{{config['DISQUS_ID']}}.disqus.com/embed.js';
+s.setAttribute('data-timestamp', +new Date());
+(d.head || d.body).appendChild(s);
+})();
+</script>
+<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
```
Comment (johnwheffner) [inline app/templates/race/details.html:344]: I haven't seen anyone use `<noscript>` in a long time. ;-)

File: app/templates/race/details.html (hunk 1 above)
Comment (jyundt) [reply app/templates/race/details.html]: Agreed, I realized I did my testing directly on a staging server and didn't commit my code.

File: app/templates/race/details.html (hunk 2 above)
Comment (jyundt) [reply app/templates/race/details.html]: Good catch.

File: app/templates/race/details.html (hunk 3 above)
Comment (jyundt) [reply app/templates/race/details.html:344]: Is this back in the day when you were writing token ring drivers?

**Review by johnwheffner — APPROVED** [review]
lgtm

**Pull request merged**